viewer/packages/tools/src/Geomap/MapConfig.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+/**
+ * Supported map API Keys
+ */
+export enum APIKeys {
+DEV_MAPBOX_API_KEY = "pk.eyJ1IjoicHJhbW9kLXMiLCJhIjoiY2tzb2JkbXdyMGd5cjJubnBrM3IwMTd0OCJ9.jA9US2D2FRXUlldhE8bZgA",
+DEV_HEREMAPS_APP_ID = "HqSchC7XT2PA9qCfxzFq",
+DEV_HEREMAPS_APP_CODE = "5rob9QcZ70J-m18Er8-rIA",
+DEV_BING_API_KEY = "AuViYD_FXGfc3dxc0pNa8ZEJxyZyPq1lwOLPCOydV3f0tlEVH-HKMgxZ9ilcRj-T",
+}
+
+/**
+ * Map Configuration of {@link GeomapTool}.
+ */
+export type MapConfig = {
```

Comment:
This type should probably be split into "common" values (such as mapProvider) and a specific config for each mapProvider. See https://github.com/cognitedata/reveal/blob/387663e45aae9c56366d5e5389dffd1722c5b20d/examples/src/pages/e2e/TestViewer.tsx#L49 for an example on how to achieve this using union types.